### ISSUE TITLE
How to remove "powered by pay.cards" from Camera

### ISSUE THREAD

**Issue by Msoniya16** [faceterteam/PayCards_iOS_Source#3]
Hi,
I am using pay. cards for scanning credit card and i want to removed "Powered by pay.card" line from camera but we don't have any access to edit camera.
I am using pods to integrate SDK in my app. So, Please let us know solution for this ASAP.

Thanks,
Shweta

**Comment by vitkuzmenko** [comment]
Hello! It possible only if you build your own framework from source code.
Hide copyrightButton in PayCardsRecognizer.mm.

**Comment by vitkuzmenko** [comment]
But you need remove simulator platform manually while archiving for AppStore.

**Comment by Msoniya16** [comment]
Thanks for the immediate response, but my issue does not resolved as i am not able to find the "PayCardRecognizer" folder to build the framework when i clone and download the code. Please find the attached screenshot.
![screen shot 2017-12-06 at 7 07 08 pm](https://user-images.githubusercontent.com/30657520/33664187-bf7a3686-dab8-11e7-9290-efbda1cf1fa8.png)

**Comment by vitkuzmenko** [comment]
<img width="1096" alt="pasted_image_06_12_2017__16_39" src="https://user-images.githubusercontent.com/3526541/33664275-14562440-daa4-11e7-813d-91b2af0ccf14.png">

**Comment by Msoniya16** [comment]
I understand that you can see that in github same as I but when i download the whole code that PayCardsRecognizer/CardRecognizer folder does not comes with the download. Might be this folder is private. Could you please check this folder after downloading the source code from github

**Comment by vitkuzmenko** [comment]
<img width="544" alt="screen shot 2017-12-06 at 16 46 50" src="https://user-images.githubusercontent.com/3526541/33664558-17795f38-daa5-11e7-809a-ac19b8b72868.png">

**Comment by Msoniya16** [comment]
Can u please share url with me for downloading source code?

**Comment by vitkuzmenko** [comment]
https://github.com/faceterteam/PayCards_iOS_Source

**Comment by Msoniya16** [comment]
Yes by this link we are able to get all the folders now we are checking and will update you in case we face any issue. 
Thanks

**Comment by Msoniya16** [comment]
Hi,

When i am run the code on xCode version 9.1 form the link (https://github.com/faceterteam/PayCards_iOS_Source) i am getting issues.  I want scanning functionality only for the ios native app. Also, i am using pods in our existing project so for this i need to use pods.
Please find the screenshot and please suggest solution for this.
![screen shot 2017-12-07 at 12 05 28 pm](https://user-images.githubusercontent.com/30657520/33701996-91d6b2ce-db47-11e7-8b98-9a512ba66bff.png)
